use_case_examples/hybrid_model/infer_hybrid_llm_generate.py

```diff
@@ -0,0 +1,89 @@
+"""Showcase for the hybrid model converter."""
+import time
+from pathlib import Path
+
+import torch
+from torch.backends import mps
+from transformers import AutoModelForCausalLM, AutoTokenizer, PreTrainedTokenizer, TextStreamer
+
+from concrete.ml.torch.hybrid_model import FHEMode, HybridFHEModel
+
+if __name__ == "__main__":
+    configs = [
+        ("transformer.h.0.mlp", 0.934),
+        (["transformer.h.0.mlp", "transformer.h.1.mlp"], 0.42),
+        ("transformer.h.0.mlp.c_proj", 0.986),
+        ("transformer.h.0.attn.c_proj", 0.986),
+    ]
+    config_index = 3
+    config = configs[config_index][0]
+
+    device = "cpu"
+    if torch.cuda.is_available():
+        device = "cuda"
+    if mps.is_available():
+        device = "mps"
+    print(f"Using device: {device}")
+
+    # Get GPT2 from Huggingface
+    # TODO: migrate to auto-model with model_name
```

Comment:
can you make an issue on this ?

Reply:
This is done already no? The following lines use the automodel

Reply:
I removed the comment